### PR TITLE
chore: downgrade Python version to 3.9

### PR DESCRIPTION
**Pull request by afiune**
## Summary

Our Python scripts doesn't need to be on version `3.11`, we are downgrading them to `3.9`
so that we can support more versions of the AWS provider.

## How did you test this change?

Updated my stack and onboarded new accounts. Scripts run successfully on the old version.

Screenshot 2023-11-17 at 10 49 29 AM

## Issue
N/A

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -143,11 +154,22 @@ resource "aws_lambda_function" "lacework_setup_function" {
...
+  runtime = "python3.9"
+
+  # Python3.10 support introduced in version 4.64.0
+  # https://github.com/hashicorp/terraform-provider-aws/blob/release/4.x/CHANGELOG.md#4640-april-20-2023
+  # runtime = "python3.10"
+
+  # Python3.11 support introduced in version 5.11.0
+  # https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md#5110-august--3-2023
+  # runtime = "python3.11"
+
```
Comment (ipcrm) [inline main.tf:172]: we want these comments left in here and above?

**Review by ipcrm — APPROVED** [review]
:shipit:

**Comment by afiune** [comment]
Screenshot 2023-11-17 at 10 57 47 AM

**Comment by afiune** [comment]
Screenshot 2023-11-17 at 11 00 26 AM

File: main.tf (hunk 1 above)
Comment (ipcrm) [reply main.tf:172]: I'm fine with them there since it's intentional! Was just asking in case it was missed unintentionally

**Pull request merged**